x-pack/plugins/apm/ftr_e2e/cypress/integration/read_only_user/service_inventory/service_inventory.spec.ts

```diff
@@ -53,6 +54,12 @@ describe('When navigating to the service inventory', () => {
     cy.visit(serviceInventoryHref);
   });
 
+  it('has no detectable a11y violations on load', () => {
+    cy.contains('Services');
```

Comment:
Checking some text on the page could be included in another test as it's not specific to accessibility (same with other files)

Reply:
It shows an error if trying to call `checkA11y()` directly

Reply:
I think `cy.contains('Services');` is needed to wait for that element to be visible before running the a11y check. Would be better if this was more obvious from the code